app/controllers/users.js

```diff
@@ -52,6 +53,9 @@ const signIn = async (req, res, next) => {
 
     logger.info(`user with email ${user.email} has logged in successfully`);
 
+    const savedToken = await tokenService.createToken(userSerialized, token);
+    logger.info(`users-controller::signIn::TokenSaved: ${JSON.stringify(savedToken)}`);
```

Comment:
Do not log tokens or security information

Reply:
Ready!